### ISSUE TITLE
Get-Probe command only lists only some probes that are

### ISSUE THREAD

**Issue by farinaanthony96** [lordmilko/PrtgAPI#219]
**Describe the bug**
*Clearly and concisely describe what you were trying to do, what happened and what you were expecting to happen*
When I run the Get-Probe command, only a small list of probes that are currently disconnected from my PRTG instance are listed. There are hundreds of other probes in my instance. This happened only after I updated to the latest API version 0.9.15.

**Steps to reproduce**
*Put the relevant code from your application that caused the issue to happen in the code block below*

```powershell
# Code goes here!
Get-Probe
```

**What is the output of `Get-PrtgClient -Diagnostic`?**

```powershell
# Output goes here!
PSVersion      : 5.1.18362.1474
PSEdition      : Desktop
OS             : Microsoft Windows 10 Enterprise
PrtgAPIVersion : 0.9.15
Culture        : en-US
CLRVersion     : .NET Framework 4.8 (528040)
PrtgVersion    : 20.4.64.1402
PrtgLanguage   : english.lng
```

**Additional context**
*Anything else I should know to help solve the issue?*
This only happened after updating to the latest version of this API. The Get-Probe command worked on the version prior to 0.9.15.

**Comment by lordmilko** [comment]
Hi @farinaanthony96,

I feel like this is most likely a duplicate of #215. Due to changes in the latest versions of PRTG, PrtgAPI had to change how it retrieves probes, however due to a bug in the latest version of PRTG that doesn't actually work as expected

If you do

```powershell
Set-PrtgClient -LogLevel All
Get-Probe -Verbose
```

in the XML response that is displayed, do you see all of the probes there?

**Comment by farinaanthony96** [comment]
No. I still only see the few disconnected probes in the XML.

**Comment by farinaanthony96** [comment]
As a work-around, I am still referencing API version 0.9.14. All of my scripts work when I reference the old API.

**Comment by lordmilko** [comment]
Hi @farinaanthony96,

Yes, I would recommend using PrtgAPI 0.9.14 for now. PrtgAPI 0.9.15 changes the way PrtgAPI retrieves probes to workaround PrtgAPI crashing due to the new "PRTG Core Server" device that is present in the latest version of PRTG, however evidently there is also a bug in PRTG which causes this workaround to not always work properly, even though it should

**Comment by farinaanthony96** [comment]
Ok. Will updating to the latest version of PRTG help with it, or will that bug still be present? I'm still running v20.

**Comment by lordmilko** [comment]
The bug is also present in the latest version of PRTG; I've reported this to Paessler, but this also raises the question of "how long as this bug been in PRTG"; potentially I may actually need to come up with a new solution for dealing with the "PRTG Core Server" device in PRTG 21.2.67, but for now I would recommend just using older versions of everything

**Comment by farinaanthony96** [comment]
Alrighty, thank you @lordmilko !

**Comment by lordmilko** [comment]
Hi @farinaanthony96,

I have pushed a pre-release version of PrtgAPI 0.9.16 which reverts the new `GetProbe` behavior of PrtgAPI 0.9.15 and implements a new workaround for the "PRTG Core Server" autonomous device issue from #215; are you able to download the latest build via the [manual](https://github.com/lordmilko/PrtgAPI/wiki/Installation#manual) installation instructions and advise whether this resolves your issue?

Regards,
lordmilko

**Comment by lordmilko** [comment]
Hi @farinaanthony96,

Are you able to advise whether the new build resolves your issue? I would like to release PrtgAPI 0.9.16 including this fix however would like to confirm the fix does indeed work properly first

Regards,
lordmilko

**Comment by lordmilko** [comment]
Hi @farinaanthony96,

Please be advised PrtgAPI 0.9.16 has now been released. To update PrtgAPI, please run Update-Module PrtgAPI and reopen PowerShell

Regards,
lordmilko
